Resfeber/View Controllers/SideMenuController.swift

```diff
@@ -0,0 +1,101 @@
+//
+//  SideMenuController.swift
+//  Resfeber
+//
+//  Created by David Wright on 11/17/20.
+//  Copyright © 2020 Spencer Curtis. All rights reserved.
+//
+
+import UIKit
+
+private let reuseIdentifer = "MenuOptionCell"
+
+enum MenuOption: Int, CustomStringConvertible, CaseIterable {
+    
+    case EditProfile
+    case LogOut
```

Comment:
Generally enum cases follow camel case syntax, meaning that these probably should start with lowercase letters.